src/Generator/ClassGenerator.php

```diff
@@ -80,8 +80,10 @@ public static function fromReflection(ClassReflection $classReflection)
         $cg->setSourceContent($cg->getSourceContent());
         $cg->setSourceDirty(false);
 
-        if ($classReflection->getDocComment() != '') {
-            $cg->setDocBlock(DocBlockGenerator::fromReflection($classReflection->getDocBlock()));
+        $docBlock = $classReflection->getDocBlock();
+
+        if ($docBlock) {
```

Comment:
```suggestion
        if ($docBlock !== false) {
```

Thoughts on this? As `$docBlock` is not entire true or false, I usually try to be specific.

Reply:
In this case, `$docBlock` is either `false` or an object, so IMO a simple conditional suffices.